clusters/app.ci/bookstack/bookstack.yaml

```diff
@@ -0,0 +1,222 @@
+  spec:
+    ports:
+    - port: 3306
+      targetPort: 3306
+      protocol: TCP
+    selector:
+      app: bookstack-db
+    type: NodePort
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: bookstack-db
+    namespace: ci
+    labels:
+      app: bookstack-db
+  spec:
+    replicas: 1
+    strategy:
+      type: Recreate
+    selector:
+      matchLabels:
+        app: bookstack-db
+    template:
+      metadata:
+        labels:
+          app: bookstack-db
+      spec:
+        containers:
+        - image: docker.io/mysql:5.7.21
+          name: bookstack-db
+          ports:
+          - containerPort: 3306
+            name: mysql
+          env:
+          - name: MYSQL_DATABASE
+            value: "bookstack"
+          - name: MYSQL_PASSWORD
+            value: "secret"
+          - name: MYSQL_ROOT_PASSWORD
+            value: "secret"
+          - name: MYSQL_USER
+            value: "bookstack"
+          volumeMounts:
+          - mountPath: /var/lib/mysql
+            name: data
+        volumes:
+        - name: data
+          emptyDir:
+            sizeLimit: 500Mi
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: bookstack
+    namespace: ci
+    labels:
+      app: bookstack
+  spec:
+    selector:
+      app: bookstack
+    ports:
+    - port: 80
+      targetPort: 80
+      protocol: TCP
+    type: LoadBalancer
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: bookstack
+    namespace: ci
+    labels:
+      app: bookstack
+  spec:
+    replicas: 1
+    strategy:
+      type: Recreate
+    selector:
+      matchLabels:
+        app: bookstack
+    template:
+      metadata:
+        labels:
+          app: bookstack
+      spec:
+        containers:
+        - name: bookstack
+          image: docker.io/solidnerd/bookstack:22.11.0
+          ports:
+          - containerPort: 80
+            name: http
+          env:
+          - name: DB_DATABASE
+            value: "bookstack"
+          - name: DB_HOST
+            value: "bookstack-db:3306"
+          - name: DB_PASSWORD
+            value: "secret"
+          - name: DB_USERNAME
+            value: "bookstack"
+          volumeMounts:
+          - mountPath: /var/www/bookstack/.env
+            subPath: .env
+            name: dot-env
+          - mountPath: /var/www/bookstack/public/uploads
+            name: public-uploads
+          - mountPath: /var/www/bookstack/storage/uploads
+            name: storage-uploads
+          - mountPath: /var/www/bookstack/storage/clockwork
+            name: storage-clockwork
+        volumes:
+        - name: public-uploads
+          emptyDir:
+            sizeLimit: 100Mi
+        - name: storage-uploads
+          emptyDir:
+            sizeLimit: 100Mi
+        - name: storage-clockwork
+          emptyDir:
+            sizeLimit: 100Mi
+        - name: dot-env
+          secret:
+            secretName: dot-env
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: dot-env
```

Comment:
This `dot-env` name is too generic. This secret will live in the `ci` namespace with hundreds (thousands?) of other, non-related, secrets so it needs to be more specific.